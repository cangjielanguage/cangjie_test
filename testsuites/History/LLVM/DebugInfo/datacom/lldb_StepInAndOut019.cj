// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_StepInAndOut019_expected.txt ../basecase/011_debug_func_reload.cj
// EXEC: cjc -g 011_debug_func_reload.cj
// EXEC: python3 DatacomTest.py lldb_StepInAndOut019_expected.txt



