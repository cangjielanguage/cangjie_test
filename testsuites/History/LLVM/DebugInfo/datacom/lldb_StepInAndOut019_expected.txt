lldb,lldb 011_debug_func_reload,011_debug_func_reload
b,b 4,011_debug_func_reload.cj:4
r,r,-> 4
s,s,-> 13
finish,finish,-> 4
n,n,-> 5
cj_plocals,cj_plocals,Functioncall008 = 10